_data/accessibility-tests/radio-buttons.yml

```diff
@@ -0,0 +1,72 @@
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 2.4.6
+  - summary: All clickable elements touch target size meet height and width requirements.
+    summary_additional: The size and spacing of the touch target are at least 24px wide and 24px tall.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 2.5.8
+# Keyboard navigation tests
+  - summary: Radio button focus can be moved and selection can be selected or deselected.
+    summary_additional: When you use the spacebar and arrow keys on a keyboard, you can toggle between selections and clearly see when the choice is selected or not.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 3.2.1
+  - summary: Radio button focus follows logical content order.
+    summary_additional: When you tab through the radio button group with a keyboard, the focus stays in order of the content as intended.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 2.4.3
+  - summary: Radio buttons are still fully functional using only keystrokes.
+    summary_additional: When you use a keyboard to navigate to the radio button option, you are able to move focus, check and uncheck buttons, and move focus into and out of the radio group. If none of the radio buttons are checked, focus is set on the first radio button in the group.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 2.1.1
+# Screen reader tests
+  - summary: Screen reader announces question before any radio button and then related elements.
+    summary_additional: When listening to the radio button using a screen reader, you hear the question related to the radio button read first. Then you will hear the radio button and label and its checked state.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.2
+    wcag_criterion: 4.1.2
+  - summary: Screen reader announces instructions for radio button fields and options for radio button groups.
+    summary_additional: When listening to the radio button using a screen reader, any instructions related to the field and you also hear all the options in the button group.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 3.3.2
+  - summary: Screen reader values for radio buttons match screen display.
+    summary_additional: When using a screen reader, you can hear the same value for the radio button that is shown on the screen.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 2.5.3
+  - summary: Screen reader announces all labels.
+    summary_additional: When you use a screen reader and navigate to a radio button, you hear every label or title associated with that radio button.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 1.3.5
```

Comment:
@amycole501 or @alex-hull 

**question:** On [checkbox](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/rc-publish-checkbox-checklist/components/checkbox/accessibility-tests/), this same rule is marked as 1.3.1. It also is tagged as conditional. Should radio match checkbox?

Reply:
Reading over that test criteria for 1.3.5 I think we can actually remove it from radio button. Input purpose is for data INPUT fields and radio buttons are not an input field. I think we also landed on input purpose being related to autocomplete in which case radio button doesn't apply. Thanks for flagging!

Reply:
Resolved in [610d52e](https://github.com/uswds/uswds-site/pull/2821/commits/610d52ee302096392b62b51d30b4551a0a179ec6). Test has been removed.

Reply:
@amycole501 

Does it matter that radio buttons _technically_ are input elements? We build them as an `<input type="radio">` elements, which means they are expected to have an associated `<label>` element. Should we reinstate this test in the checklist? 

Reference:
- [MDN input type="radio](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio)
- [USWDS radio button code sample](https://designsystem.digital.gov/components/radio-buttons/#component-code)

Reply:
I believe that the SC refers to when a person enters personal data like first name or last name. I do think we can remove that check since the SC in question is to ensure a person can use autofill quickly to identify input's purpose.